apps/chat/src/components/Settings/CustomLogoSelect.tsx

```diff
@@ -18,6 +19,8 @@ interface CustomLogoSelectProps {
   className?: string;
   fileManagerModalTitle?: string;
   allowedTypes?: string[];
+  disabledState?: boolean;
+  disabledTooltip?: string;
```

Comment:
Please, use common names for component properties without specifying
```suggestion
  disabled?: boolean;
  tooltip?: string;
```